examples/dem/3d-rotating-drum/post_processing_rotating_drum.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from lethe_pyvista_tools import *
+from numpy.ma.core import arctan
+from numpy.ma.extras import average
+
+# Experimental date extracted using Web plot digitizer
+df_x_zero_exp = pd.read_csv("./data_exp.csv", header=None,
+                            names=['Column1', 'Column2'])
+
+# Path where the parameter file and the output folder is
+path = "."
+
```

Comment:
Can you use the argparser module instead to specify the path instead of hardcoding it? You can look at the TGV example for how it is used.